finetune/base_models/bert/modeling.py

```diff
@@ -779,11 +796,6 @@ def transpose_for_scores(
     # seem a bit unusual, but is taken from the original Transformer paper.
     attention_probs = dropout(attention_probs, attention_probs_dropout_prob)
 
```

Comment:
After all this we need to move things back around so elements appear at the same spots

Reply:
Which we can do by reshaping to add heads again, seperately splitting, the reshaping back and concatting